src/foam/nanos/auth/Subject.js

```diff
@@ -89,6 +89,15 @@ foam.CLASS({
       javaCode: `
         return getUser().getId() != getRealUser().getId();
       `
+    },
+    {
+      name: 'toSummary',
+      type: 'String',
+      javaCode: `
+        return isAgent() ?
```

Comment:
Can this not just use toString above?  As it outputs the entire path.

Reply:
That's a good question. 

Would we want to have toSummary potentially reference a long list of user summaries? Example being "john doe >> jane doe >> john doe >> cookie doe co". This approach can extend a considerable amount I'd imagine and already has a place in the toString method.

I've always thought of toSummary on an object to provide a legible description that includes the objects top interest property values. In my uses of the subject I haven't cared much for the user path.